### PR TITLE
Update type of 'log_data' from blob to longblob

### LINKED ISSUE #1022: log_data should be a LONGBLOB
### Component

agent

### Describe the bug

`logs.log_data` should be a LONGBLOB or greater: particularly long log lines (>65kb) causes issues with mysql's default BLOB type used by xorm for byte arrays: manually altering the table fixed the issue for me, but it would be probably better to alter the type for everyone :)
I don't use pipeline artifacts, but I bet the same problem would be present in files bigger than 65kb when using mysql.

### System Info

```shell
`woodpeckerci/woodpecker-agent:next` on arm64
```

### Additional context

_No response_

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

### PR DESCRIPTION
**Pull request by simmstein**
fix #1022

### CODE REVIEW

**Comment by 6543** [comment]
@simmstein only a migration is missing, would you mind to add it?

https://woodpecker-ci.org/docs/development/guides#add-a-new-migration

if you need help please tell us

**Comment by simmstein** [comment]
First try in go-lang :) I had the migration task.

File: server/store/datastore/migration/006_log_data_type.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package migration
+
+import (
+	"xorm.io/xorm"
+	"xorm.io/xorm/schemas"
+)
+
+var alterTableLogUpdateColumnLogDataType = task{
+	name: "alter-table-logs",
```
Comment (6543) [inline server/store/datastore/migration/006_log_data_type.go:9]:
```suggestion
	name: "alter-table-logs-update-type-of-data",
```
bit more unique

**Review by 6543 — APPROVED** [review]
Thanks :)

**Comment by 6543** [comment]
@simmstein would you mind backporting it?

-> just cherry-pick the merge commit onto release branch

**Pull request merged**